### PR TITLE
address PR comments

### PR DESCRIPTION
**Pull request by ytaous**
**Description**: address PR comments

**Motivation and Context**
- https://github.com/microsoft/onnxruntime/pull/3174#discussion_r397046004
- https://github.com/microsoft/onnxruntime/pull/3174#discussion_r397047086
- https://github.com/microsoft/onnxruntime/pull/3174#discussion_r397048438
- https://github.com/microsoft/onnxruntime/pull/3174#discussion_r396853069
- https://github.com/microsoft/onnxruntime/pull/3174#discussion_r396859643
- https://github.com/microsoft/onnxruntime/pull/3174#discussion_r397008895
- https://github.com/microsoft/onnxruntime/pull/3174#discussion_r397681772

### CODE REVIEW

File: onnxruntime/core/framework/execution_frame.cc [hunk 1]
```diff
@@ -222,8 +222,8 @@ ExecutionFrame::ExecutionFrame(const std::vector<int>& feed_mlvalue_idxs, const
                              : nullptr;
           buffers_[mem_patterns_->locations[i]] = BufferUniquePtr(buffer, alloc);
 
-          // comment out following line to see the size of activation
-          // printf("Allocated memory for activations, size: %zu\n", mem_patterns_->patterns[i].PeakSize());
+          // log size of activation
+          VLOGS(session_state_.Logger(), 1) << "Allocated memory for activations, size: " << mem_patterns_->patterns[i].PeakSize();
```
Comment (SherlockNoMad) [inline onnxruntime/core/framework/execution_frame.cc]: Please keep this commented out for now... 
Otherwise, this will flood the log...

Comment (ytaous) [reply onnxruntime/core/framework/execution_frame.cc]: ok

Comment (edgchen1) [reply onnxruntime/core/framework/execution_frame.cc]: VLOG is already for verbose debug logging, so i think flooding is ok. i kind of agree with the original comment.

File: onnxruntime/core/framework/execution_frame.cc [hunk 2]
```diff
@@ -343,8 +343,11 @@ Status ExecutionFrame::AllocateMLValueTensorPreAllocateBuffer(OrtValue& ort_valu
 
     // be generous and use the buffer if it's large enough. log a warning though as it indicates a bad model
     if (buffer_num_elements >= required_num_elements) {
-      // Todo: View Operator is reusing the buffer bigger than the required size. Disabling warming message for now.
-      //LOGS_DEFAULT(WARNING) << message;
+      // View Operator is reusing the buffer bigger than the required size. 
+      // Disabling warning message for now. The op is in the process of being deprecated.
+#ifndef ENABLE_TRINING
```
Comment (edgchen1) [inline onnxruntime/core/framework/execution_frame.cc]: ENABLE_TRINING -> ENABLE_TRAINING

Comment (ytaous) [reply onnxruntime/core/framework/execution_frame.cc]: good catch

**Pull request merged**